cnx-contrib/src/widgets/cpu.rs

```diff
@@ -72,8 +72,7 @@ impl Cpu {
             + (current.nice_time - previous.nice_time)
             + (current.system_time - previous.system_time)
             + (current.idle_time - previous.idle_time)
-            + (current.iowait_time - previous.iowait_time)
-            + (current.total_time - previous.total_time);
```

Comment:
Any specific reason for removing `total_time` ? IIRC, I ported the algorithm directly from xmobar.

Reply:
The `total_time` is `user_time` + `nice_time` + `system_time` as you can see on line 138. That means `user_time`, `nice_time` and `system_time` is added twice for `diff_total`.
As a result the calculated percentage on line 79 was only half as high as it should have been.